libs/wallet-ui/src/index.tsx

```diff
@@ -35,6 +37,19 @@ const ROOT = 'wallet-ui'
  * Renders all the providers
  */
 export function App({ service, client, runtime, features }: AppProps) {
+  useEffect(() => {
+    if ('Cypress' in window) {
+      Globals.assign({
+        skipAnimation: true,
+      })
+    }
```

Comment:
Don't we need `else` here? It looks like in case of `cypress` it assigns `skipAnimation: true` but then it assigns skip Animation to `true` again.

Reply:
This is just so the animations are off for screenshots so we don't get time differences, it's set to false by default so we only want to enable it if cypress is running. The return function is cleanup for when the hook unmounts (pretty much never) but we want to leave the app in a consistent state with animation on when tearing down.